packages/host/modules/clusterio/compat.lua

```diff
@@ -148,9 +150,14 @@ if major_v1_1 then
 	-- Can not use ... here because of luals type suggestion breaking when it is used
 	-- TODO maybe include a version that does not require game?
 
-	compat.table_to_json = function(tbl) return game.table_to_json(tbl) end
-	compat.json_to_table = function(json) return game.json_to_table(json) end
-	compat.write_file = function(filename, data, append, for_player) return game.write_file(filename, data, append, for_player) end
+	if game ~= nil then
+		compat.table_to_json = function(tbl) return game.table_to_json(tbl) end
+		compat.json_to_table = function(json) return game.json_to_table(json) end
+		compat.write_file = function(filename, data, append, for_player) return game.write_file(filename, data, append, for_player) end
+	else
```

Comment:
This block will never be ran because `game` is only present during runtime and the file cannot be required at runtime. Therefore we should either always use `json.encode` and `json.decode` for 1.1 or we move the check inside of the function calls such as:
```lua
compat.table_to_json = function(tbl)
    if game then
        return game.table_to_json(tbl)
    end
    return json.encode(tbl)
end
```